skunkworks_crow/src/main/AndroidManifest.xml

```diff
@@ -56,7 +64,13 @@
         </activity>
         <activity
             android:name=".views.ui.hotspot.HpSenderActivity"
-            android:configChanges="orientation|screenSize" />
+            android:configChanges="orientation|screenSize"
+            android:parentActivityName=".views.ui.send.SendFormsActivity"
+            >
+            <meta-data
```

Comment:
No need to add metadata for this feat

Reply:
Did the needful.

Reply:
I can understand that, can you rephrase that

Reply:
Removed configChanges for now. It wasn,t needed for this commit.

Reply:
I still can see in the file changes can you please check that

Reply:
it's not there for me, shall I send a new pr?

Reply:
No need to do new PR you still can work on this

Reply:
please run `./gradlew checkcode` locally